includes/Core/Modules/Modules.php

```diff
@@ -566,7 +566,18 @@ protected function get_registry() {
 	 * @return Module_Registry
 	 */
 	protected function setup_registry() {
-		$registry          = new Module_Registry();
+		$registry = new Module_Registry();
+		/**
+		 * Filters an array with the list of all the available modules slugs, each slug present on this array would
+		 * be registered for inclusion. If a module is `force_activate` it would be included even if the module is
```

Comment:
Very tiny issue

Looking through the repository, the search-console and site-verification are set to `force_active`, not `force_activate`.

Reply:
Looks like a bad on my side it should be `force_active` .